packages/react-components/src/components/utils/filtrer-tree/sort.tsx

```diff
@@ -69,3 +82,33 @@ export const sortNodes = (nodes: TreeNode[], sortConfig: SortConfig[]): TreeNode
     }
     return sortedNodes;
 };
+
+const parseValue = (valueString: string): number | undefined => {
+    let floatNumber = NaN;
+    let factor = -1;
+    const valueArray = valueString.split(' ');
+    if (valueArray.length === 1) {
+        floatNumber = Number.parseFloat(valueString);
+        factor = 1;
+    } else if (valueArray.length === 2) {
+        const value = valueArray[0];
+        const unit = valueArray[1];
+        if (unit === 'ns' || unit === '%') {
```

Comment:
It's probably OK as this is only a temporary fix, but it seems that for comparing a unit-less number with a unit-suffixed number, the factors should be 1E-9 / 1E-6 / 1E-3 / 0.01 for ns / us / ms / %.

Reply:
not necessary... if it's unit less then we don't know what the value represent. They could be ms. So, mixing different units will lead incorrect sorting. This should be fixed once a fullstack solution is implemented.